server/config/config.js

```diff
@@ -3,7 +3,8 @@ export const FIRESTORE_CONFIG_ROOT = '';
 
 export const ETH_CONTRACT_ADDRESS = '0x1111111111111111111111111111111111111111';
 
-export const COSMOS_ENDPOINT = 'http://LIGHT_CLIENT_ENDPOINT/';
+export const COSMOS_LCD_ENDPOINT = 'http://LIGHT_CLIENT_ENDPOINT/';
```

Comment:
Do we still need to keep the `COSMOS_LCD_ENDPOINT`? Seems no files are going to use it?